bin/tma_commander.py

```diff
@@ -130,31 +130,18 @@ def __init__(self, host, log_level, simulate):
         )
         self.command_loop_task = asyncio.create_task(self.command_loop())
         self.command_dict = dict(
-            az_power=MTMount.commands.AzimuthAxisPower,
```

Comment:
Are you sure you want to remove these commands? I think it is ok to keep them around in the commander, gives us a chance that check what happens if we issue them.

Reply:
I'm confused. Why would we want to try to send commands to elevation and azimuth? I guess it's not actively harmful, but it seems strange.

Reply:
I'll restore all commands. I guess it's a bit simpler having the file identical to the main branch.